pyproject.toml

```diff
@@ -6,12 +6,14 @@ authors = ["LocalNewsLab <mh3287@columbia.edu>"]
 license = "MIT"
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = ">=3.9, <3.11"
```

Comment:
🔧 Gotta keep this to python3.9, nothing higher, because AWS lambda runtimes do not support python beyond 3.9. I'll make a quick change to the CI

Reply:
Can we do `~3.9` instead? When I tried to install pandas type stubs for mypy it wasn't happy with `^3.9`.